import-dashboards.py

```diff
@@ -242,6 +242,25 @@ def add_datasources(api_key):
             print ' * Cannot add PTSummary Data Source'
             sys.exit(-1)
 
+    if 'Prometheus AlertManager' not in ds:
+        print ' * Prometheus AlertManager Data Source'
+        data = json.dumps({
+            'name': 'Prometheus AlertManager',
+            'type': 'camptocamp-prometheus-alertmanager-datasource',
+            'url': 'http://localhost:9093/alertmanager/',
+            'access': 'proxy',
+            'basicAuth': False,
+            'jsonData': {'keepCookies': []},
```

Comment:
It could be just `'jsonData': {}` as above, so the future reader does not have to think what is so special with `[]` there…

Reply:
^